docs/en/end-user/expose.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Expose Application Publicly
+---
+
+TBD, Content Overview：
+
+1. add service/ingress/route trait.
```

Comment:
Please choose from svc ingress route.

Reply:
I would like to choose ingress if only one